beetmoverscript/templates/firefox_partner.yml

```diff
@@ -0,0 +1,42 @@
+---
+metadata:
+    name: "Beet Mover Manifest"
+    description: "Maps Firefox artifacts to pretty names for en-US"
+    owner: "release@mozilla.com"
+
+s3_bucket_path: pub/firefox/candidates/{{ version }}-candidates/build{{ build_number }}/
```

Comment:
Hm, is the private partners path different? If so, it would be nice to be able to reuse the template, but change the `s3_bucket_path` if the private flag is set.